2021-04-13 16:15:56,850 INFO    MainThread:14028 [wandb_setup.py:_flush():69] setting env: {}
2021-04-13 16:15:56,850 INFO    MainThread:14028 [wandb_setup.py:_flush():69] setting login settings: {}
2021-04-13 16:15:56,850 INFO    MainThread:14028 [wandb_init.py:_log_setup():319] Logging user logs to C:\Users\Jaimie\Documents\VU\Master jaar 1\Data Mining Techniques\DataMiningTechniques\wandb\run-20210413_161556-o013b56h\logs\debug.log
2021-04-13 16:15:56,851 INFO    MainThread:14028 [wandb_init.py:_log_setup():320] Logging internal logs to C:\Users\Jaimie\Documents\VU\Master jaar 1\Data Mining Techniques\DataMiningTechniques\wandb\run-20210413_161556-o013b56h\logs\debug-internal.log
2021-04-13 16:15:56,851 INFO    MainThread:14028 [wandb_init.py:init():352] calling init triggers
2021-04-13 16:15:56,852 INFO    MainThread:14028 [wandb_init.py:init():359] wandb.init called with sweep_config: {}
config: {}
2021-04-13 16:15:56,852 INFO    MainThread:14028 [wandb_init.py:init():401] starting backend
2021-04-13 16:15:56,852 INFO    MainThread:14028 [backend.py:_multiprocessing_setup():71] multiprocessing start_methods=spawn, using: spawn
2021-04-13 16:15:56,858 INFO    MainThread:14028 [backend.py:ensure_launched():123] starting backend process...
2021-04-13 16:15:57,321 INFO    MainThread:14028 [backend.py:ensure_launched():128] started backend process with pid: 33496
2021-04-13 16:15:57,322 INFO    MainThread:14028 [wandb_init.py:init():406] backend started and connected
2021-04-13 16:15:57,322 INFO    MainThread:14028 [wandb_init.py:init():446] updated telemetry
2021-04-13 16:15:57,323 INFO    MainThread:14028 [wandb_init.py:init():465] communicating current version
2021-04-13 16:15:58,851 INFO    MainThread:14028 [wandb_init.py:init():470] got version response upgrade_message: "wandb version 0.10.25 is available!  To upgrade, please run:\n $ pip install wandb --upgrade"

2021-04-13 16:15:58,851 INFO    MainThread:14028 [wandb_init.py:init():478] communicating run to backend with 30 second timeout
2021-04-13 16:15:59,190 INFO    MainThread:14028 [wandb_init.py:init():503] starting run threads in backend
2021-04-13 16:15:59,889 INFO    MainThread:14028 [wandb_run.py:_console_start():1422] atexit reg
2021-04-13 16:15:59,891 INFO    MainThread:14028 [wandb_run.py:_redirect():1285] redirect: SettingsConsole.WRAP
2021-04-13 16:15:59,892 INFO    MainThread:14028 [wandb_run.py:_redirect():1320] Wrapping output streams.
2021-04-13 16:15:59,892 INFO    MainThread:14028 [wandb_run.py:_redirect():1336] Redirects installed.
2021-04-13 16:15:59,892 INFO    MainThread:14028 [wandb_init.py:init():527] run started, returning control to user process
2021-04-13 16:16:01,029 INFO    MainThread:14028 [wandb_run.py:_atexit_cleanup():1392] got exitcode: 1
2021-04-13 16:16:01,030 INFO    MainThread:14028 [wandb_run.py:_restore():1364] restore
2021-04-13 16:16:01,253 INFO    MainThread:14028 [wandb_run.py:_wait_for_finish():1515] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 926
  total_bytes: 926
}

2021-04-13 16:16:03,264 INFO    MainThread:14028 [wandb_run.py:_wait_for_finish():1515] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 4309
  total_bytes: 4309
}

2021-04-13 16:16:04,463 INFO    MainThread:14028 [wandb_run.py:_show_files():1737] logging synced files
